source/PyMaterialX/PyMaterialXCore/PyElement.cpp

```diff
@@ -14,8 +14,8 @@
 
 #define BIND_ELEMENT_FUNC_INSTANCE(T)                                                                           \
 .def("_addChild" #T, &mx::Element::addChild<mx::T>)                                                             \
-.def("_getChildOfType" #T, &mx::Element::getChildOfType<mx::T>)                                                 \
-.def("_getChildrenOfType" #T, &mx::Element::getChildrenOfType<mx::T>, py::arg("category") = mx::EMPTY_STRING)   \
+/*.def("_getChildOfType" #T, &mx::Element::getChildOfType<mx::T>)                                             */\
+/*.def("_getChildrenOfType" #T, &mx::Element::getChildrenOfType<mx::T>, py::arg("category") = mx::EMPTY_STRING) */\
```

Comment:
@jstone-lucasfilm @kwokcb Does this need new python implementation here
https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/python/MaterialX/main.py#L39

Reply:
Perhaps we can just start with the proposed C++ API changes, and once these are locked down, we can move on to the Python and JavaScript bindings?